docs/reference/generators.rst

```diff
@@ -0,0 +1,61 @@
+.. _generators:
+
+===================================
+Generators and Application Modeling
+===================================
+
+.. currentmodule:: dimod.generators
+
+.. automodule:: dimod.generators
+
+Benchmarking
+============
```

Comment:
Physics maybe?

Reply:
They are not really physics applications, rather than benchmarking problems developed by physicists. Put another way,tThey use physics language, but their purpose is benchmarking.